src/bp-templates/bp-nouveau/js/buddypress-activity-post-form.js

```diff
@@ -76,6 +78,92 @@ window.bp = window.bp || {};
 			this.postFormPlaceholder.inject( '#bp-nouveau-activity-form-placeholder' );
 		},
 
+		checkDraftContentInPostForm: function(){
+			var self = this;
+
+			var hasDraft = false;
+
+
+			var $whatsNew = self.postForm.$el.find( '#whats-new' );
+
+
+			$( document ).on( 'paste keyup change', '#whats-new', function(){
+
+				if ( self.postForm.$el.hasClass( 'bp-activity-edit' ) ){
+					return;
+				}
+
+				var $postContent = $( $.parseHTML( window.group_messages_editor.getContent() ) );
+				var postContent = $postContent.text();
+				var hasEmoji = $whatsNew.find( 'img.emojioneemoji' ).length;
+
+				if ( postContent.length || hasEmoji > 0 ){
+					hasDraft = true;
+				}else{
+					hasDraft = false;
+				}
+
+				self.toggleDraftClass( hasDraft );
+
+			});
+
+		},
+
+		checkDraftAttachments: function(){
+
+			if ( this.postForm.$el.hasClass( 'bp-activity-edit' ) ){
+				return;
+			}
+
+			var hasDraft;
+
+			var $whatsNewAttachments = $( '#whats-new-attachments' );
+			var countAttachmets = $whatsNewAttachments.find( '.dz-preview' ).length;
```

Comment:
@shohelbuddyboss can you check this without `dz-preview` class elements length and make use of dropzone objects to find if any media and document is added or not in the object?